.github/workflows/release.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       with:
         before_all: yarn prepack
         npm_publish: yarn publish
-        ignore: packages/website packages/convergence packages/interactor packages/cli
+        ignore: packages/website packages/cli
```

Comment:
Why are we ignoring `cli`?

Reply:
Hmmm I'm not sure. I was wondering about that too. I checked to see who added it and turns out I added it from the get go ([PR 132](https://github.com/thefrontside/bigtest/pull/132/)). 🤷 

I'll go ahead and remove it.